tests/helpers/patch-reducer.js

```diff
@@ -0,0 +1,41 @@
+/* global require, define */
+
+import originalReducer from 'dummy/reducers/index';
+
+const { unsee } = require;
+
+export function applyPatch() {
+  unsee('dummy/services/redux');
+  unsee('dummy/reducers/index');
+
+  define('dummy/reducers/index', ['exports', 'dummy/reducers/low', 'dummy/reducers/high', 'redux'], function (exports, _low, _high, _redux) {
+    'use strict';
+
+    Object.defineProperty(exports, "__esModule", {
```

Comment:
Did you find the need for this `__esModule` definition? I think it's not required with the loader ember-cli uses, but might be mistaken on best practices.